build/make/variables.mk

```diff
@@ -36,6 +36,8 @@ endif
 YARN_TARGET=$(WORKDIR)/node_modules
 BOWER_TARGET?=$(WORKDIR)/public/vendor
 
+APT_REPO?=ces
```

Comment:
This can not be defined here as it may get changed in a future makefiles release. This has to be defined in the main Makefile.